test/check-ostree

```diff
@@ -41,25 +41,15 @@ CHECKOUT_LOCATION = "/var/local-tree"
 RPM_LOCATION = "/usr/share/rpm"
 KEY_ID = "95A8BA1754D0E95E2B3A98A7EE15015654780CBD"
 
-# old cockpit/ws container on rhel-atomic still has "[ ] Reuse my password for privileged tasks"
-login_opts = {}
-if os.environ.get("TEST_OS") == "rhel-atomic":
-    login_opts["legacy_authorized"] = True
-
 
 def get_list_item(index):
     # there are more <li> inside the item tables, so only count direct children
     return ".available-deployments > li:nth-child({0})".format(index)
 
 def get_os_name(m):
-    if m.image == "rhel-atomic":
-        return "rhel-atomic-host"
-    elif m.image == "fedora-coreos":
+    if m.image == "fedora-coreos":
         return "fedora-coreos"
-    elif 'continuous' in m.image:
-        return 'centos-atomic-host'
-    else:
-        return "fedora-atomic"
+    return "fedora-atomic"
```

Comment:
Fedora Atomic also hasn't been a thing for a while. Want to clean that up as well while you are at it? I think this should rather `raise NotImplementedError` here.

Reply:
I was wondering what should be the fallback. Is there any chance that we will introduce anything else? If not, then I can just replace calls to this function with directly adding `fedora-coreos` into the code.

Reply:
Right now there's only fedora-coreos. RHEL coreos isn't a standalone supported product. We may add this at some point, but as there is nothing planned right now, I'd just make it as simple as possible, and we can reintroduce an indirection if/when we need it.